Examples/encoder.c

```diff
@@ -583,18 +621,17 @@ int main (int argc, char **argv) {
 		fclose(fp2);
 	}
 
+	/* Calculate rate in MB/sec and print */
+	timing_set(&t2);
+	tsec = timing_delta(&t1, &t2);
+	printf("Encoding (MB/sec): %0.10f\n", (((double) size)/1024.0/1024.0)/totalsec);
+	printf("En_Total (MB/sec): %0.10f\n", (((double) size)/1024.0/1024.0)/tsec);
 
```

Comment:
Why did you change the place where the code block for calculating rate was ( before placing after freeing allocated memory ) ?